napari/_app_model/context/_context_keys.py

```diff
@@ -0,0 +1,20 @@
+from typing import TYPE_CHECKING, Generic, TypeVar
+
+from app_model.expressions import ContextNamespace as _ContextNamespace
+
+if TYPE_CHECKING:
+    from ...utils.events import Event
+
+A = TypeVar("A")
+
+
+class ContextNamespace(_ContextNamespace, Generic[A]):
+    """A collection of related keys in a context
+
+    meant to be subclassed, with `ContextKeys` as class attributes.
```

Comment:
```suggestion
    meant to be subclassed, with class attributes that are `ContextKeys`.
```

I was a little confused on this point. I was looking at LLCK as an example and expected an attribute _named_ "ContextKeys".